src/test/java/org/mule/tools/apikit/MainAppScaffolderWithExistingConfigApiSyncTest.java

```diff
@@ -36,15 +46,45 @@ public void reScaffold() throws Exception {
     ResourceLoader testScaffolderResourceLoader = new TestScaffolderResourceLoader(TEST_RESOURCES_APISYNC + "/v1");
     ApiReference apiReference = ApiReference.create(RAML_RESOURCE_URL_V1, testScaffolderResourceLoader);
 
-    ScaffoldingResult result = scaffoldApi(RuntimeEdition.CE, apiReference);
+    ScaffoldingResult result = scaffoldApi(RuntimeEdition.CE, apiReference, null, null, null);
     verifySuccessfulScaffolding(result, TEST_RESOURCES_APISYNC + "/v1/api.xml");
 
     List<MuleConfig> muleConfig = new ArrayList<>(result.getGeneratedConfigs());
-    ScaffoldingResult secondScaffoldingResult = scaffoldApi(RuntimeEdition.CE, apiReference, muleConfig);
+    ScaffoldingResult secondScaffoldingResult = scaffoldApi(RuntimeEdition.CE, apiReference, null, muleConfig, null);
 
     verifySuccessfulScaffolding(secondScaffoldingResult, TEST_RESOURCES_APISYNC + "/v1/api.xml");
   }
 
+  @Test
+  public void testRescaffoldSameVersionsGlobals() throws Exception {
```

Comment:
I still don't get the point of this test. Would you mind to write down a Javadoc?